how-to-docs/amazon-resources.md

```diff
@@ -0,0 +1,55 @@
+# How to get Amazon Resources in the UK Cluster
+
+## Workflow
+1. Pull stacks-hod-platform repo
```

Comment:
The repo will be `acp-<clustername>-infra-resources`, so for example: `acp-prod-infra-resources`